### PR TITLE
Igal/mgmt 1347

### PR DESCRIPTION
**Pull request by tsorya**

### CODE REVIEW

File: src/commands/step_processor.go [hunk 1]
```diff
@@ -85,6 +86,10 @@ func (s *stepSession) processSingleSession() int64 {
 	result, err := s.Client().Installer.GetNextSteps(s.Context(), &params)
 	if err != nil {
 		s.Logger().Warnf("Could not query next steps: %s", err.Error())
+		if reflect.TypeOf(err) == reflect.TypeOf(installer.NewGetNextStepsNotFound()) {
```
Comment (ori-amizur) [inline src/commands/step_processor.go:89]: I think checking the type of the error and relying on the type of the error in order to decide what to do is not a good idea.
It is not part of the protocol that 404 for next-steps is sleep forever.  If someone will see it he will not know why it was done.

Comment (tsorya) [reply src/commands/step_processor.go:89]: that's what i was asked too

Comment (tsorya) [reply src/commands/step_processor.go:89]: i don't think u need to remind to say the truth. It is not interesting to anyone. 
Or we are sending from bm-inventory sleep forever or just verifying the error here.
It ticket i was asked to verify the error, it is easier but we can open a dispute one more time.

Comment (ori-amizur) [reply src/commands/step_processor.go:89]: OK. Sent this also in the ticket.  If this opinion will persist then we will go on with it.

Comment (ori-amizur) [reply src/commands/step_processor.go:89]: @ronniel1 FYI

Comment (avishayt) [reply src/commands/step_processor.go:89]: I think this is fine and expected.  If the cluster no longer exists, there is nothing for it to do.

**Pull request merged**